backend/src/main/java/com/redhat/cloud/notifications/db/repositories/EndpointRepository.java

```diff
@@ -82,7 +81,7 @@ public Endpoint createEndpoint(Endpoint endpoint) {
     public List<Endpoint> getEndpointsPerCompositeType(String orgId, @Nullable String name, Set<CompositeEndpointType> type, Boolean activeOnly, Query limiter) {
 
         Query.Limit limit = limiter == null ? null : limiter.getLimit();
-        Query.Sort sort = limiter == null ? null : limiter.getSort();
+        Optional<Query.Sort> sort = limiter == null ? Optional.empty() : limiter.getSort();
```

Comment:
It's already set by caller - This is the part that is tricky, because we could set them here again and override whatever we set in there. I don't recall (will check) if this is the case, but there could be multiple callers with different sorting needs / options. This is sometimes enforced by the query itself (e.blabla.foo vs blabla.foo vs foo, i.e. the relations used in the query)

Reply:
Really? I thought this was called by `EndpointResource` (and maybe other places) and that class doesn't appear in the modified files in this PR.

Reply:
Let me double check :grin:

Reply:
They were not set anywhere, thanks for noticing!